newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -176,14 +179,26 @@ export const HomePage = React.memo<Props>(
       // If the user is not authenticated, the GetStarted section is displayed.
       React.useEffect(
         () => {
-          setActiveTab(authenticated ? initialTab : 'get-started');
+          if (shouldChangeTabAfterUserLoggedIn.current) {
+            setActiveTab(authenticated ? initialTab : 'get-started');
+          }
         },
         // Only the initialTab at component mounting is interesting
         // and we don't want to change the active tab if initialTab changes.
         // eslint-disable-next-line react-hooks/exhaustive-deps
         [authenticated]
       );
 
+      // Once the home page is mounted, do not automatically change tab unless
+      // the user is logging in (this should only happen when the editor is opening).
+      React.useEffect(
+        () => {
+          if (loginState === 'loggingIn') return;
```

Comment:
so this doesn't trigger a redirect when a user logs in or signs up on the get started page?

Reply:
I'll add a better comment